components/webcompat_reporter/browser/webcompat_report_uploader.cc

```diff
@@ -100,8 +127,8 @@ void WebcompatReportUploader::CreateAndStartURLLoader(
             "Though the 'Report a Broken Site' option of the help menu or"
             "the Brave Shields panel."
           data: "Broken URL, IP address, Shields settings, language settings,"
-                "Brave VPN connection status; user provided additional details"
-                "and contact information."
+                "Brave VPN connection status, user-provided additional details,"
+                "optional screenshot and contact information."
           destination: OTHER
           destination_other: "Brave developers"
         }
```

Comment:
it's not really specific to this PR, but i noticed this feature is available in Tor. it should therefore use a profile-specific URL loader factory so that the Tor page report also goes through Tor, otherwise we should warn users they are submitting a request that won't be through Tor. https://github.com/brave/brave-browser/wiki/Simple-Guide-to-SimpleURLLoader#deciding-on-a-urlloaderfactory

Reply:
It's definitely much better to have the report go through Tor. If we can't do that now, we should open a ticket to fix it.

Reply:
Actually, on further thought, we should probably disable the feature if it's not going through Tor.

Reply:
> it should therefore use a profile-specific URL loader factory so that the Tor page report also goes through Tor, otherwise we should warn users they are submitting a request that won't be through Tor

fortunately, we already do this

https://github.com/brave/brave-core/blob/master/browser/ui/webui/webcompat_reporter/webcompat_reporter_ui.cc#L71-L72

Reply:
seems fine to me. @arthuredelstein good to close this comment?

Reply:
Yes, sounds good